prog/heartbeat.rb

```diff
@@ -5,7 +5,10 @@
 class Prog::Heartbeat < Prog::Base
   label def wait
     DB["SELECT 1"].first
-    Excon.get(Config.heartbeat_url)
+    Excon.get(Config.heartbeat_url, read_timeout: 2, write_timeout: 2, connect_timeout: 2)
+    nap 10
+  rescue Excon::Error::Timeout => e
+    Clog.emit("heartbeat request timeouted") { {exception: {message: e.message}} }
```

Comment:
timeouted -> timed out

Reply:
Thanks. Interesting ChatGPT didn't fix it for me